src/api/api.client.js

```diff
@@ -0,0 +1,482 @@
+import { normalize, schema } from 'normalizr';
```

Comment:
This is the old `api/index.js` file, with some additions in the end : The client api.middleware uses to query the REST api and paginate the results